lib/matplotlib/contour.py

```diff
@@ -198,8 +226,10 @@ def clabel(self, levels=None, *,
             print('End manual selection with second mouse button.')
             if not inline:
                 print('Remove last label by clicking third mouse button.')
-            blocking_contour_labeler = BlockingContourLabeler(self)
-            blocking_contour_labeler(inline, inline_spacing)
+            mpl._blocking_input.blocking_input_loop(
+                self.axes.figure, ["button_press_event", "key_press_event"],
+                timeout=-1)(functools.partial(
+                    _contour_labeler_handler, self, inline, inline_spacing))
```

Comment:
The `partial` can be passed directly as last argument, no?

Reply:
yes